_posts/2019-09-10-guide_building_the_game.md

```diff
@@ -47,41 +46,50 @@ It may also be called `origin/develop`:
 </div>
 
 ## Windows
-Download Momentum Mod on Steam or grab the `MomentumDev` folder from [***Google Drive***](https://drive.google.com/file/d/115xT5IIN-CimKCZQho3xj7uIcYFMSZXm/view).
+Download Momentum Mod on Steam, or grab the `MomentumDev` folder from [***Google Drive***](https://drive.google.com/file/d/115xT5IIN-CimKCZQho3xj7uIcYFMSZXm/view).
 
 Download [***Visual Studio 2019***](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community&rel=16).
 
+<div class="note info">
+    <p>
+When installing Visual Studio, make sure this version is checked:
+
+<img src="/assets/images/build_guide/build6.png" alt="" style="display: block; margin: auto;">
```

Comment:
This image does not give a lot of context on where people can find it. I would suggest using a bigger picture that shows this tool box highlighted.